### ISSUE TITLE
Improve discussion of accessibility in explainer

### ISSUE THREAD

**Issue by ltenmoz** [w3c/edit-context#136]
The EditContext explainer currently implies that using EditContext improves accessibility. This is true to a certain extent with the non-`<canvas>` EditContext, but it should be made clearer that

- With `<canvas>` elements, EditContext *does not* help with accessibility at all
- Even with non-canvas EditContext, care still has to be taken to ensure that the editor's DOM contents are accessible

See https://github.com/mozilla/standards-positions/issues/199#issuecomment-3226968951 - there is some concern that with the current language, developers may assume that simply by using EditContext, their editors automatically become accessible, which is not the case.

In theory we could expose the `EditContext.text` to assistive technologies, to resolve these issues instead (would probably require some kind of `ScreenCoordinateToTextPosition` event for querying where in the text a certain screen position lies). Although, I'm not sure if that would work in all cases, for example, it wouldn't expose any text formatting, which might be important to certain assistive technologies. Furthermore, some editors such as Google Docs don't store all of the document's text in `EditContext.text` (only the part around the caret).
